minifi/README.md

```diff
@@ -0,0 +1,163 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+      https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+# Apache NiFi -  MiNiFi [![Build Status](https://travis-ci.org/apache/nifi-minifi.svg?branch=master)](https://travis-ci.org/apache/nifi-minifi)
```

Comment:
seems to me this file should go away and the readme of nifi should be updated to reflect that one might want to create/use the convenience binary of minifi instead of nifi.

Reply:
Most of NiFi's README applies to MiNiFi as well (you still have to build the whole thing, plus mailing lists and such are the same), so I just created a section in NiFi's README for the MiNiFi subproject that refers to the binaries, Docker images, etc.